adagrams/game.py

```diff
@@ -1,11 +1,159 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+
+    my_list = [] #creating an empty list to collect randomly picked letter
+
+    #converting dictionary keys into list representing the 'count' times of letter and storing in pool_list
+    pool_list = []
+    for letter, count in LETTER_POOL.items():
+        pool_list += [letter] * count # add letter 'count' times to pool_list
+        
+    while len(my_list) < 10:
+        random_key = random.randint(0, len(pool_list) - 1) #randomly select index from pool_list
```

Comment:
`random_key` doesn't quite capture what the value it references is. What does `randint` return and what does that value represent?

Since the function returns an integer that represents a random index we'll use to get a random letter, `random_key` could be renamed to something like `random_index`.